### ISSUE TITLE
Feature Request - Add ability to set password upon launching application

### ISSUE THREAD

**Issue by JoshuaJones21** [jimradford/superputty#700]
Ideally, this would prevent unauthorised users from accessing the application and potentially gaining access to machines, this would act as another layer of security.

**Comment by afranks-luminex** [comment]
The password protection should really be on the underlying services/keys - superputty's design purpose is an interface into various terminal emulators, not a gatekeeper. If I've misunderstood your feature request, please provide a specific example of what you're describing.

**Comment by JoshuaJones21** [comment]
Yes, that is fair. I've seen this on other applications & I think it acts nicely as another layer of security.

**Comment by NYCDoITT** [comment]
Greetings,

PuTTY Connection Manager v0.7.1 BETA (build 136) keeps its sessions in an encrypted database, that can be password protected. 

Do you plan on implementing this functionality, or at least encrypting the Sessions.XML?

Regards,
